en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -124,6 +124,9 @@ The Enforcer component in Choreo Connect can log analytics-related data to be us
      
     1. Compile the new custom reporter implementation as a JAR file.</br>
     2. Mount it to the `/home/wso2/lib/dropins` directory within the Enforcer by adding the JAR file to the choreo-connect-1.x.x/docker-compose/resources/enforcer/dropins directory.</br>
+
+        !!! note 
+            If Choreo is Connect deployed using the Kubernetes Helm artifacts, dropins folder mounting to the Enforcer is explained in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-dropins-directory-to-a-choreo-connect-helm-deployment)
```

Comment:
```suggestion
            Helm Chart users, please refer to the documentation [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-dropins-directory-to-a-choreo-connect-helm-deployment) to add a JAR file into the Droppings directory.
```

Reply:
Please use the updated doc link if the doc header is updated with the other comments. I mean `#mount-dropins-directory-to-a-choreo-connect-helm-deployment` part of the link